bigtop_toolchain/bin/puppetize.sh

```diff
@@ -69,7 +69,8 @@ case ${ID}-${VERSION_ID} in
         puppet module install puppetlabs-stdlib --version 4.12.0
         # Enabling the PowerTools and EPEL repositories via Puppet doesn't seem to work in some cases.
         # As a workaround for that, enable the former here in advance of running the Puppet manifests.
-        dnf config-manager --set-enabled crb
+        dnf config-manager --set-enabled crb devel
```

Comment:
This should be done in the [Puppet manifest](https://github.com/apache/bigtop/blob/master/bigtop-deploy/puppet/modules/alluxio/manifests/init.pp)? We can not expect that the puppetize.sh is applied in all deployment.